Dockerfile

```diff
@@ -6,6 +6,9 @@ COPY . $HOME/application
 
 WORKDIR $HOME/application/wellregistry
 
+RUN apt-get update
+RUN apt-get install build-essential libpq-dev python3-dev -y
+
```

Comment:
Not a blocker to a merge, but I'd recommend combining these into a single `RUN`.

Reply:
Might also be worth trying `apt-get install gcc` instead of build-essential.

Reply:
thank you, I wondered about that myself. I am also looking at placing a remove/purge after it is finished to see if that makes a difference. I will amend the PR with these ideas to trim the size down.

Reply:
As to the first comment, I have them combined in my workspace while I work on a trimming. I submitted before I thought if merging them because I thought we needed a rapid response to a broken build. Also, I presumed they could be combined like the CMD but had not confirmed it until after. I appreciate the suggestions here because they confirm my suspicions.